lms/djangoapps/onboarding/forms.py

```diff
@@ -420,8 +416,7 @@ def save(self, request, commit=True):
             user_info_survey.start_month_year = None
             user_info_survey.hours_per_week = 0
             user_info_survey.role_in_org = None
-            selected_function_areas = ""
-            user_info_survey.user.extended_profile.save_user_function_areas(selected_function_areas)
+            user_info_survey.user.extended_profile.save_user_function_areas(list())
```

Comment:
Do we really need this now?
As we are using model form and single field to handle function_areas